app/controllers/dev_controller.rb

```diff
@@ -46,6 +46,9 @@ def reset
             user:
           )
         end
+
+      Team.find_by(ods_code: "Y51") ||
+        FactoryBot.create(:team, name: "NMEPFIT SAIS Team", ods_code: "Y51")
```

Comment:
Is it better not to seed this?

Reply:
The issue is that the dev controller's `/reset` truncates the `teams` table. The example campaign creates a team so it's not  as simple as changing `/reset` to leave it, I think we should 🥾🥫 this and worry about it later.